### PR TITLE
docs: added the cart page

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Cart Page.

image

### CODE REVIEW

File: stylesheets/cart.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.quantity-options{
+    display: flex;
+    gap: 10px;
+    align-items: center;
+}
+
+.quantity-options>button{
+    display: flex;
```
Comment (notmafiaa) [inline stylesheets/cart.css:60]: it will work without selector also

File: stylesheets/cart.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    align-items: center;
+    justify-content: center;
+    width: 20px;
+    height:20px;
+    border: thin solid;
```
Comment (notmafiaa) [inline stylesheets/cart.css]: px values , any specific reason you have used it other than rems?

**Review by notmafiaa — APPROVED** [review]
Overall Code Looks Great 

can be merged!!

**Review by j24m — APPROVED** [review]
Checked  every line , coudn't really find anything that needs change, except in line where you used 12 px, so if that was on purpose you can ignore this, if not you can change it. Your UI is really beautiful, good choice of colors.<3

File: stylesheets/cart.css (hunk 2 above)
Comment (SinchanaMS) [reply stylesheets/cart.css]: I missed it. Thank you for pointing it out. :)

**Comment by SinchanaMS** [comment]
> Checked every line , coudn't really find anything that needs change, except in line where you used 12 px, so if that was on purpose you can ignore this, if not you can change it. Your UI is really beautiful, good choice of colors.<3

Thank you so much, Jyotsna! :)

**Pull request merged**